web/src/components/Login.js

```diff
@@ -0,0 +1,71 @@
+import PropTypes from 'prop-types';
+import { useState } from 'react';
+import { Input, Row, Col, Button, Typography, Layout } from 'antd';
+
+import React from "react";
+
+import { LoginOutlined, EyeInvisibleOutlined, EyeTwoTone } from '@ant-design/icons';
+import 'antd/dist/antd.css';
+
+const LoginButtonStyle = {
+  backgroundColor: '#00c853',
+  borderColor: '#00c853'
+};
+
+const { Text } = Typography;
+
+const Login = () => {
+    const [username, setUsername] = useState('')
+    const [password, setPassword] = useState('')
+  
+    const login = () => {
+      console.log('Trying to log in with', username, password);
+    }
+  
+    const onKeyPress = (e) => {
+      if (e.charCode == 13) {
+        login();
+      }
+    }
+    return (
+        <Row gutter={[0, 16]}>
+        <Col span={24}></Col>
+        <Col span={6} offset={9}>
+          <Text strong>Username or Email</Text>
+          <Input
```

Comment:
It would be better if we encapsulated these <Input> components with a <Form> component. <Form> components provide extra useful functionalities regarding input handling.

Reply:
Thank you, I will implement the changes and commit them.